firebase-dynamic-links/src/test/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.google.firebase.dynamiclink.test"
+          android:versionCode="1"
+          android:versionName="1">
+
+    <uses-sdk android:minSdkVersion="16"/>
+
+    <application>
+        <service android:name="com.google.firebase.components.ComponentDiscoveryService">
+            <meta-data
+               android:name="com.google.firebase.components:com.google.firebase.analytics.connector.FakeConnectorComponent"
```

Comment:
I can't see this class in your PR, is it needed?
If not, you can delete this manifest, no?